datahub-web-react/src/app/entity/container/ContainerEntity.tsx

```diff
@@ -99,9 +98,9 @@ export class ContainerEntity implements Entity<Container> {
                 {
                     component: SidebarDomainSection,
                 },
-                {
-                    component: SidebarRecommendationsSection,
-                },
+                // {
+                //    component: SidebarRecommendationsSection,
+                // },
```

Comment:
should we just delete here and other places? or we could leave a comment as to why we we're leaving it